extension/test/e2e_visual_test.ts

```diff
@@ -133,6 +133,29 @@ describe('Visual Regression Tests', function () {
     });
   });
 
+  describe('zoom', function () {
+    for (const level of [1, 2, 3, 4]) {
+      it(`should work at zoom level ${level}`, async function () {
+        const clock = sinon.useFakeTimers();
+        const span = insertHtmlIntoDomAndReturnFirstTextNode(
+          '<span>虫眼鏡</span>'
+        ) as HTMLSpanElement;
+        await triggerMousemoveAtElementStart(span);
```

Comment:
Super nit, but can you add a newline before this line and after line 149 to separate the 'parts' of the test similar to the other tests. (It's a small boon to see arrange, act, and assert sections independently)